### ISSUE TITLE
Error: Code 404

### ISSUE THREAD

**Issue by dhumann** [isekaidev/stable.art#78]
The plugin recently stopped working and began throwing 404 errors.

Yes. The URL is correct and --api is enabled. I've seen numerous other people having the same issue.  

![image](https://github.com/isekaidev/stable.art/assets/841236/9ee740fd-13de-41c8-84b0-de7d6077808d)

**Comment by SeBL4RD** [comment]
Same problem here.

**Comment by emiliyaukhanova** [comment]
Have you resolved this problem? I've got the same((

**Comment by donaldafeith** [comment]
This problem is not with the plugin. After pulling, re-pulling, I finally decided to try an older version of Stable Diffusion.

Guess what? It's nothing to do with the plugin. Stable Diffussion (I'm going to assume everyone here is using Automatic111 WebUI? Might be wrong...) But at the moment of writing this... There have been changes that completely break the hooks on this plugin.

**Comment by dhumann** [comment]
> Have you resolved this problem? I've got the same((

I have not resolved it. Other SD-Photoshop plugins still work so I'm not sure what's up with this one. 

I've tried it with both A1111 and SD-Forge and neither work with Stable.art

**Comment by SeBL4RD** [comment]
Personally, it works with my Forge. The problem is probably that they had to change something in the function protocol with the API. It's been updated with the other plugins, but not with this one.
I've got an old version of Forge that must be 2 months old, that certainly explains why it still works with it.

**Comment by w-e-w** [comment]
- most likely it's caused by https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/15333
this splits the sampler and scheduler into two different fields
which means if you are still using the sampler plus scheduler name for the scheduler argument then it will [404](https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/1c0a0c4c26f78c32095ebc7f8af82f5c04fca8c0/modules/api/api.py#L43-L48)

we have two compatibility PRs pending
- https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/15656
- https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/15681

---

actions you can take now
1. get someone make an update to the API client if possible
2. only use samplers without the scheduler name the
3. downgrade to webui 1.8

**Comment by emiliyaukhanova** [comment]
> * most likely it's caused by [Scheduler selection in main UI AUTOMATIC1111/stable-diffusion-webui#15333](https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/15333)
>   this splits the sampler and scheduler into two different fields
>   which means if you are still using the sampler plus scheduler name for the scheduler argument then it will [404](https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/1c0a0c4c26f78c32095ebc7f8af82f5c04fca8c0/modules/api/api.py#L43-L48)
> 
> we have two compatibility PRs pending
> 
> * [Allow old sampler names in API AUTOMATIC1111/stable-diffusion-webui#15656](https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/15656)
> * [more old sampler scheduler compatibility AUTOMATIC1111/stable-diffusion-webui#15681](https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/15681)
> 
> actions you can take now
> 
> 1. get someone make an update to the API client if possible
> 2. only use samplers without the scheduler name the
> 3. downgrade to webui 1.8

Yes, I've got it. Now in Photoshop I use samplers without the sheduler name and it works ok.
